meetings/2021-02-10.md

```diff
@@ -0,0 +1,96 @@
+* Review Minutes
+    * Accept [prior minutes](https://github.com/solid/authorization-panel/pull/165/files?short_path=77842b3#diff-77842b35acf47a1c2030c48d0b1e6d626b9dc3c9e07d168d265815b1fdbd41f2) (2m)
+    * Accept [minutes for Monday's special session](https://github.com/solid/authorization-panel/pull/167/files?short_path=e66d5b0#diff-e66d5b08e777fcb77d751f43b35a40f779e28776d19fde98601f334c1105e56a) (4 min)
+* Pull Requests    
+    *  https://github.com/solid/authorization-panel/pull/166 - Add create and delete modes
+    *  https://github.com/solid/authorization-panel/pull/152 - Add functional requirements   
+* Topic Items
+    * Use cases and requirements target - 2/17 (Next Weds)
+        * Discuss outstanding items / #166 and #152 to get there
+
+## Minutes
+
+Justin: Looking to see if we can finish the UCR for next week. 
+
+Action Item: Any remaining reviews must be complete by Friday
+
+Elf: wondering for background of #166 to help review by Friday.
+
+Justin: One of the problem is that it bundles the ability to create and delete into write and append and creation of a resource. So one can overpermission. One could add triples to container. 
+
+Henry: I'd just add: speaking in terms of delete and create makes the UCR easier to read.
+
+Pavlik: Consider adjusting "write" to "update"
+
+Pavlik: does append really make sense?
+
+Justin: for logging yes. Perhaps we need a use case for that.
+
+Justin: Proposing to rename document "Use Cases and Requirements for Authorization in Solid"
+
+Justin: Will also change path to "authz-ucr"
+
+min 30: 
+  Just leaves early for personal commitment.
+
+## Discussion on Credentials 
+
+Henry: CSarven [announced on gitter](https://gitter.im/solid/specification?at=6022bab5063b6c68d54c658c) that 
+> Meeting with the CredentialsCG is fixed to March 10 at 15:00 UTC and 18:00 UTC (two ~1h sessions) - seems to work for most. CCG presents to SocialCG in the first session, and the other way around in the second session.
+
+Henry: as Credentials are part of my EU funded [Solid Control Project](https://nlnet.nl/project/SolidControl/) I found this to be a great opportunity to learn more about it. 
+
+Henry: Having gone over the UCR I have now time to dedicate to Credentials. So I opened an issue [Web Credentials Wallet questions](https://github.com/solid/authentication-panel/issues/126) on authN panel, but it could have just as well have been here. It was very helpful to watch Drummond Reed's talk from 4 months ago on the Self-Sovereign project, as it puts a lot of pieces into context. This lead me to download the Manning book [Self Sovereign Identity](https://www.manning.com/books/self-sovereign-identity) that is nearly finished. I think this is invaluable, as it helps put a lot of the different players, technologies and arguments in that space into context. This would be nearly impossible to get an overview of by just reading specifications. So I highly recommend it.
```

Comment:
```suggestion
Henry: Having gone over the UCR I have now time to dedicate to Credentials. So I opened an issue [Web Credentials Wallet questions](https://github.com/solid/authentication-panel/issues/126) on authN panel, but it could have just as well have been here. It was very helpful to watch Drummond Reed's talk from 4 months ago on the Self-Sovereign project, as it puts a lot of pieces into context. This led me to download the Manning book [Self Sovereign Identity](https://www.manning.com/books/self-sovereign-identity) that is nearly finished. I think this is invaluable, as it helps put a lot of the different players, technologies and arguments in that space into context. This would be nearly impossible to get an overview of by just reading specifications. So I highly recommend it.
```